lib/animina_web/live/profile_stories_live.ex

```diff
@@ -263,7 +263,7 @@ defmodule AniminaWeb.ProfileStoriesLive do
           >
             <.live_component
               module={AniminaWeb.StoryComponent}
-              id={"story_#{story.id}"}
+              id={"story_#{story.id}_#{story.created_at}"}
```

Comment:
We talked about this. This is the wrong approach. The id should be unique. If it isn’t we have a bug somewhere else in the application. That has to be fixed. Contact @wintermeyer in case you need any open heart surgery in the production database for old entries.